lib/features/modeling/BpmnUpdater.js

```diff
@@ -341,11 +345,34 @@ BpmnUpdater.prototype.updateParent = function(element, oldParent) {
   this.updateDiParent(di, parentDi);
 };
 
+// Handle embedded labels with defined bounds, to stay compatible with other vendors
+function shapeHasExplicitlyBoundedInternalLabel(shape) {
+  return (is(shape, 'bpmn:Activity'))
+    && getDi(shape).label
+    && getDi(shape).label.bounds != undefined;
```

Comment:
I believe it's enough to check if bounds are truthy (they should always be an object):

```suggestion
    && getDi(shape).label.bounds;
```